hoot-test/src/main/cpp/hoot/test/main.cpp

```diff
@@ -78,15 +78,15 @@ typedef boost::shared_ptr<CppUnit::Test> TestPtr;
 
 enum _TestType
 {
-  CURRENT,
-  QUICK,
-  QUICK_ONLY,
-  SLOW,
-  SLOW_ONLY,
-  GLACIAL,
-  GLACIAL_ONLY,
-  SERIAL,
-  ALL
+  CURRENT       = 0x01,
+  QUICK_ONLY    = 0x02,
+  QUICK         = 0x03,
+  SLOW_ONLY     = 0x04,
+  SLOW          = 0x07,
+  GLACIAL_ONLY  = 0x08,
+  GLACIAL       = 0x0f,
+  SERIAL        = 0x10,
+  ALL           = 0x1f
```

Comment:
Converted these into bit flags: `SLOW == SLOW_ONLY | QUICK` etc.  This allowed for the simplifications below.

Reply:
I <3 this.